condition.go

```diff
@@ -117,3 +117,37 @@ func Lte(column string, value interface{}) Builder {
 		return buildCmp(d, buf, "<=", column, value)
 	})
 }
+
+// Lk is `LIKE`.
+// When value is nil, do nothing.
+// When value is a slice, do nothing.
+// Otherwise it will be translated to `LIKE`.
+func Lk(column string, value interface{}) Builder {
+	return BuildFunc(func(d Dialect, buf Buffer) error {
+		if value == nil {
+			return nil
+		}
+		v := reflect.ValueOf(value)
+		if v.Kind() == reflect.Slice {
```

Comment:
it is not good to hide errors. need to return error or pass value as is.
I think array of runs is valid argument for  like, but it has type slice.